nens_auth_client/urls.py

```diff
@@ -25,6 +25,33 @@
 except ImportError:  # Django 1.11 compatibility
     from django.conf.urls import url as re_path
 
+
+def override_admin_auth(admin_path="admin"):
+    """This should be included in the urlpatterns of the root site before
+    the admin urls are included.
+
+    The login and logout paths are overriden. An admin/local-login/ path is
+    added for backdoor access with local accounts.
+    """
+    return [
+        re_path("^{}/login/".format(admin_path), views.login, name="admin-login-override"),
```

Comment:
Ik heb wel eens naar `{% url 'login' %}` ofzo verwezen. Idem met `urlreverse('login')`. Dat daat dan, vrees ik, nog naar de oude login pagina's? Of gaat dat wel goed?

Reply:
De `login` is geloof ik door lizard-auth-client gedefinieerd. Admin zou `admin:login` moeten zijn en drf `rest_framework:login`.